### ISSUE TITLE
UI not loaded and displayed

### ISSUE THREAD

**Issue by waomodder** [liasece/sd-webui-train-tools#4]
sd-webui: commit: 22bcc7be checkpoint: 8838e0d1fb
sd-webui-train-tools: ver e60b0486
Python: ver 3.10.6
The UI is always loading and doesn't display properly, even when extensions are added.
The UI only shows the passage of time.
Also, disabling other extensions does not cure this infinite loading problem.
![image](https://user-images.githubusercontent.com/67697932/228533171-a5cae5ec-c492-41c2-9f0f-0fdd65900f5c.png)

**Comment by liasece** [comment]
It looks like a webui issue:
https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/22bcc7be

**Comment by kingzeus** [comment]
同样的问题，几乎不可用了，有临时修复的方案么？

**Comment by liasece** [comment]
> 同样的问题，几乎不可用了，有临时修复的方案么？

https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/22bcc7be

**Comment by ThatObjective9** [comment]
> > 同样的问题，几乎不可了，有临時修复的案么?
> 
> [AUTOMATIC1111/stable-diffusion-webui@ 22bcc7be](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/22bcc7be)

I can't use train tools with 22bcc7b as well as waomodder. I'm not very knowledgeable and don't understand what you mean by linking above.
Does that mean that I will be able to use train tools if I work with reference to this link?
Or does it mean that train tools can't handle it because it's a webui cause?

**Comment by MiDark** [comment]
When I added argument `--no-gradio-queue` to the startup of stable-diffusion-webui, the UI load success. 
It is not clear what other effects, you can try it.

**Comment by ThatObjective9** [comment]
Thank you for your response. I will try the option you suggested.

**Comment by ThatObjective9** [comment]
> When I added argument `--no-gradio-queue` to the startup of stable-diffusion-webui, the UI load success. It is not clear what other effects, you can try it.

Thank you for your advice. I added the option and now the UI is displaying fine. I'll take a look at this for a while.

**Comment by Rokurolize** [comment]
> When I added argument `--no-gradio-queue` to the startup of stable-diffusion-webui, the UI load success. It is not clear what other effects, you can try it.

I also encountered a similar issue, but I was able to resolve it with this method!

Reproduction steps:
1. Create a project
2. Create a version
3. Upload an image
4. Click on `Update Dataset`
5. After the process is complete, restart the WebUI
6. Upon opening the `Train Tools` tab, the UI elements show an infinite loading display

By adding `--no-gradio-queue` when launching, it displayed correctly.

My environment:
python: 3.10.10  •  torch: 2.0.0+cu118  •  xformers: 0.0.19.dev516  •  gradio: 3.23.0  •  commit: [22bcc7be](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/22bcc7be428c94e9408f589966c2040187245d81)
